apiary.apib

```diff
@@ -4854,6 +4854,77 @@ with the table are returned in the response.
                 }
             ]
 
+### Create or Update with Columns [POST /v2/storage/tables/{table_id}/metadata]
```

Comment:
Vůbec bych to jako with columns neuváděl je to update metadat tabulky, dal bych to v dokumentaci před ten původní a u toho bych napsal deprecated.

Reply:
OK, upravim. Bral jsem to jako novou moznost...

Reply:
Spíš náhrada tych formú sa chceme zbavit